lib/authz/project_update_manager.go

```diff
@@ -1,23 +1,18 @@
-package projectupdater
+package authz
```

Comment:
This is the main change. There are two new interfaces EsClient and ConfigManager. The EsClient allows the start, stop, and status of the elasticsearch jobs. The ConfigManager allows the retrieving and storing of the stage status.